src/sql/workbench/contrib/charts/browser/utils.ts

```diff
@@ -0,0 +1,35 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IChartsConfiguration } from 'sql/workbench/contrib/charts/browser/interfaces';
+import * as nls from 'vs/nls';
+import { IConfigurationService } from 'vs/platform/configuration/common/configuration';
+import { INotificationService, Severity } from 'vs/platform/notification/common/notification';
+import { IStorageService, StorageScope, StorageTarget } from 'vs/platform/storage/common/storage';
+
+/**
+ * Gets the max allowed row count for chart rendering.
+ */
+export function getChartMaxRowCount(configurationService: IConfigurationService): number {
+	return configurationService.getValue<IChartsConfiguration>('charts').maxRowCount;
```

Comment:
Would suggest making a const for this ID so we don't accidently update it and forget to change this